frontend/src/components/workspace/UserOrganizationInviteClientWrapper.tsx

```diff
@@ -0,0 +1,28 @@
+"use client";
```

Comment:
if we remove the feature flag check, this component doesn't seem useful unless I'm missing something